### PR TITLE
app-emulation/img: EAPI bump, fix bugs and other minor changes

### PR DESCRIPTION
**Pull request by linxon**
Closes: https://bugs.gentoo.org/699564
Closes: https://bugs.gentoo.org/679282
Package-Manager: Portage-2.3.79, Repoman-2.3.16
Signed-off-by: Yury Martynov <email@linxon.ru>

### CODE REVIEW

File: app-emulation/img/img-0.5.7-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+EGO_PN="github.com/genuinetools/img"
+
+inherit golang-vcs-snapshot
+
+DESCRIPTION="Standalone daemon-less unprivileged Dockerfile and OCI container image builder"
+HOMEPAGE="https://github.com/genuinetools/img"
+SRC_URI="https://github.com/genuinetools/img/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="MIT"
```
Comment (juippis) [inline app-emulation/img/img-0.5.7-r1.ebuild]: Remember that these packages need to tell licenses of bundled deps as well. You can use `go-license-detector` to get a grasp what those might be for this package.

File: app-emulation/img/img-0.5.7-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+DEPEND="seccomp? ( sys-libs/libseccomp )"
+RDEPEND="${DEPEND}
+	app-emulation/runc"
+
+src_compile() {
+	GOPATH="${S}:$(get_golibdir_gopath)" \
+		GOCACHE="${T}/go-cache" \
+		IMG_DISABLE_EMBEDDED_RUNC=1 \
+		go build -v -work -x -tags "noembed $(usev seccomp)" \
```
Comment (juippis) [inline app-emulation/img/img-0.5.7-r1.ebuild]: `usex`?

File: app-emulation/img/img-0.5.7-r1.ebuild (hunk 1 above)
Comment (linxon) [reply app-emulation/img/img-0.5.7-r1.ebuild]: /tmp/portage/app-emulation/img-0.5.7-r1/work/img-0.5.7/src/github.com/genuinetools/img
```
	99%	MIT
	84%	MIT-feh
```

File: app-emulation/img/img-0.5.7-r1.ebuild (hunk 2 above)
Comment (linxon) [reply app-emulation/img/img-0.5.7-r1.ebuild]: https://devmanual.gentoo.org/eclass-reference/ebuild/index.html
```
usev <USE item>
    Like use, but also echoes USE item when use returns true. 
```

Comment (juippis) [reply app-emulation/img/img-0.5.7-r1.ebuild]: Oh, sweet! Wasn't aware of this.

**Pull request closed without merging**